submissions/franqly/franqly.md

```diff
@@ -1,25 +1,24 @@
+The fastest speedrunner in the world - at time of writing this - is a newish player called Franqly. He joined the Factorio speedrunner community recently with his first entry on [speedrun.com](https://www.speedrun.com/user/Franqly) only in October last year. However, today he is out-running legends like AntiElitz and Nefrums, and is the only player alive that has single-handedly finished Factorio in [90 minutes flat](https://www.speedrun.com/factorio/run/y2xelq5m). In the time it takes you to watch the Bee Movie he can complete everything from crash landing on Nauvis to launching the rocket.
 
-It's no suprise then that Franqly was recently accepted to take part in the [Summer Games Done Quick](https://gamesdonequick.com/), or SGDQ for short, which is happening from 4 to 11 July 2021. This event is an enormous yearly speedrunning marathon which showcases over [140 different runs](https://gamesdonequick.com/tracker/runs/SGDQ2021). It even sports various big names such as [TheMexicanRunner](https://www.speedrun.com/user/themexicanrunner) and [mitchflowerpower](https://www.speedrun.com/user/mitchflowerpower). And it's for a great and noble cause too! The goal of this marathon to raise funds for [Doctors Without Borders](https://www.doctorswithoutborders.org/), for which it's **the largest fundraising event in the world!** Being accepted into SGDQ is no small feat either - this year the event received almost [2000 submissions](https://gamesdonequick.com/submission/all) of which only about 150 is accepted to take part.
+It's no suprise then that Franqly was recently accepted to take part in the [Summer Games Done Quick](https://gamesdonequick.com/), or SGDQ for short, which is happening from 4 to 11 July 2021. This event is an enormous yearly speedrunning marathon which showcases over [140 different runs](https://gamesdonequick.com/tracker/runs/SGDQ2021). It even sports various big names such as [TheMexicanRunner](https://www.speedrun.com/user/themexicanrunner) and [mitchflowerpower](https://www.speedrun.com/user/mitchflowerpower). And it's for a great and noble cause too! The goal of this marathon to raise funds for [Doctors Without Borders](https://www.doctorswithoutborders.org/), for which it's **the largest fundraising event in the world!** Being accepted into SGDQ is no small feat either - this year the event received almost [2000 submissions](https://gamesdonequick.com/submission/all) of which only about 150 is accepted to take part. Talented Factorio players have submitted their runs in the hope to join this prestigeous event in the past, but was denied because Factorio was not "officially released".
```

Comment:
I'm not sure if there were submissions in the past or why there were rejected, just that this specific GDQ had multiple Factorio submissions (can be seen when searching on the submission list). So the last sentence might need a fact check.

Reply:
We have a source that reached out to Clown, who said he and Rain tried to submit, and that's the reason they were given :)